Makefile

```diff
@@ -86,6 +86,9 @@ install:
 	@echo "${APP} installed into ${INSTALLPATH}"
 
 ## lint: runs a number of code quality checks against the source code
+cilint:
```

Comment:
I should have mentioned, I called this `cilint` just so I could leave the original `lint` target in. Its a poor name, given that ci doesn't use it, it just uses the action. Oops.

I think the `cilint` target should be renamed `lint`, and the (current) `lint` target deleted.